tests/unit/challenges/test_urls.py

```diff
@@ -0,0 +1,74 @@
+from challenges.models import Challenge
```

Comment:
please move this set of import statements after rest framework import. Order should be like

```
from django.contrib......
from django.urls..........

from rest_framework.test ...........

from challenges.models ........
from hosts.models .........
from participants.models ........
```